src/components/DateRangePicker.jsx

```diff
@@ -441,7 +447,7 @@ export default class DateRangePicker extends React.Component {
           })}
           attachment={`top ${anchorDirection}`}
           targetAttachment={`bottom ${anchorDirection}`}
-          offset="-23px 0"
+          offset={`-${portalMarginTop}px 0`}
```

Comment:
We're going to want some logic somewhere to hide the caret when the container gets too close to the input I think.